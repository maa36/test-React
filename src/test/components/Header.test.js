import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header/Header';

test('should test Header component', () => {
  const wrapper = shallow(<Header name="test" />);
  expect(wrapper).toMatchSnapshot();
});