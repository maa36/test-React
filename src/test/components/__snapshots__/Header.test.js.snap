// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Header component 1`] = `
<Fragment>
  <div
    className="d-flex align-items-center justify-content-between"
  >
    <h2
      className="h3"
    >
      test
    </h2>
    <Link
      to="/add"
    >
      Add product
    </Link>
  </div>
  <hr />
</Fragment>
`;
